### ISSUE TITLE
Move `TimeStampedModel`, `User` and `UserAndContactCommonInfo` data models to the `common` app

### ISSUE THREAD

**Issue by Sepehr-Sobhani** [bcgov/cas-registration#1877]
#### Describe the task

A clear and concise description of what the task is.

#### Acceptance Criteria

- [ ] first
- [ ] second
- [ ] third

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by hannavovk** [comment]
@patriciarussellCAS We are pulling this in for Next Sprint Backlog on the Moose side

**Comment by patriciarussellCAS** [comment]
@Sepehr-Sobhani wondering if this is still needed? If so, we could pull it into a sprint this Q. Thoughts?

**Comment by Sepehr-Sobhani** [comment]
@patriciarussellCAS Yes to both questions. This is an old piece of tech debt, and it would be good to tackle it.
There is one catch though: moving the `User` data model might be a bit tricky. It could require a lot of code changes and take a significant amount of time to implement. I think we should time-box it. If it turns out to be a much bigger change than expected, we can skip that part for now and only move `TimeStampedModel` and `UserAndContactCommonInfo` tables while leaving the `User` data model as is.

**Comment by zoeyli-46** [comment]
Refinement notes: Andrea: can be pushed, its not as high priority as RLS because RLS is for security and this is for code-cleanliness
